heron/packing/tests/java/com/twitter/heron/packing/binpacking/FirstFitDecreasingPackingTest.java

```diff
@@ -148,21 +178,23 @@ public void testDefaultContainerSizeWithPadding() throws Exception {
 
     Assert.assertEquals(packingPlan.containers.size(), 2);
 
-    long totalRam = (spoutParallelism + boltParallelism + 1) * instanceRamDefault
-        + (50 * (spoutParallelism + boltParallelism) * instanceRamDefault) / 100;
+    long totalRam = (spoutParallelism + boltParallelism + HERON_INTERNAL_CONTAINERS)
+        * instanceRamDefault
+        + (padding * (spoutParallelism + boltParallelism) * instanceRamDefault) / 100;
 
     Assert.assertEquals(packingPlan.resource.ram, totalRam);
 
     double totalCpu = Math.round(spoutParallelism * instanceCpuDefault
-        + (50 * spoutParallelism * instanceCpuDefault) / 100)
+        + (padding * spoutParallelism * instanceCpuDefault) / 100)
```

Comment:
This is a bit of a nit, but we divide by 100 for the padding right? In that case whenever we do that could we write this as `padding / 100` to make it more clear why the 100 exists?